### ISSUE TITLE
GO15VENDOREXPERIMENT is default for go 1.6

### ISSUE THREAD

**Issue by guilhem** [joefitzgerald/go-plus#420]
Currently I have to set env `GO15VENDOREXPERIMENT` but this isn't necessary for go 1.6

Having same behavior for this plugin will be really helpful and will really help to use some tools like [_glide_](https://github.com/Masterminds/glide)

**Comment by joefitzgerald** [comment]
I use go-plus with glide and the vendor experiment with go 1.6 and it works just fine. Are you sure you don't have a 1.5.x runtime sitting on your PATH? Try running `which -a go` if you're on OS X or Linux.

**Comment by guilhem** [comment]
I have a _go 1.6.1_ from Ubuntu Xenial.
_go-plus_ is working if I'm setting `GO15VENDOREXPERIMENT` but this isn't necessary anymore.

**Comment by joefitzgerald** [comment]
What is the path to your go executable? Is it at /usr/local/go/bin/go? What's the output from go version?

**Comment by guilhem** [comment]
```sh
$ which -a go
/usr/bin/go
$ /usr/bin/go version
go version go1.6.1 linux/amd64
```

**Comment by joefitzgerald** [comment]
What's the output from `go env`?

**Comment by joefitzgerald** [comment]
(Ideally in a terminal where you haven't set the vendor experiment env var).

**Comment by guilhem** [comment]
```
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/guilhem/go:/home/guilhem"
GORACE=""
GOROOT="/usr/lib/go-1.6"
GOTOOLDIR="/usr/lib/go-1.6/pkg/tool/linux_amd64"
GO15VENDOREXPERIMENT="1"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0"
CXX="g++"
CGO_ENABLED="1"
```
quite strange... I have `GO15VENDOREXPERIMENT` :)
(I didn't know `go env`)

**Comment by joefitzgerald** [comment]
OK, and just to rule out any possibilities, you have no output at all from `/usr/local/bin/go env` or `/usr/local/go/bin/go env`, right?

**Comment by joefitzgerald** [comment]
Have you built glide from source? When did you last update glide itself?

**Comment by guilhem** [comment]
I haven't any `/usr/local/bin/go/` or `/usr/local/go/` at all :)

`glide` is from last week (with a `go get`) and is working perfectly fine from command line.
`go build` too.

I only have problem with _go-plus_ to find vendor libs.

**Comment by joefitzgerald** [comment]
That's really weird. Can we screenshare in about an hour and a half? If so, hop on Slack (badges are at the top of the go-plus README) and join #go-plus and DM me.

**Comment by guilhem** [comment]
If it can help, when I'm launching _atom_ without any env, it isn't working (vendor libs aren't find).
If I'm opening a new _atom_ with `GO15VENDOREXPERIMENT=1 atom" it's working perfectly fine.

**Comment by joefitzgerald** [comment]
Does https://github.com/joefitzgerald/go-plus/issues/386#issuecomment-199359955 help at all?

**Comment by guilhem** [comment]
@joefitzgerald I'm using _fish_ or I'm launching atom with _unity launcher_

But I try to use your comment (editing profile and bashrc) and using bash:
```
$ bash
$ atom
```

And I have another behavior :/
* My vendored libs aren't found
* Others tests are working

**Comment by joefitzgerald** [comment]
This issue is fixed in go 1.7. Additionally, on Ubuntu - unless you are proficient with editing the relevant files to affect the environment for unity launcher launched processed, I would suggest just launching atom from the terminal.

If you are having issues with vendored libs not being found, try running `gocode close`.
